jenkins/bootstrap_test.py

```diff
@@ -1311,6 +1311,18 @@ def Check(job, name):
             'job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml',
             Check, use_json=is_modern)
 
+    def testBootstrapSecurityPullYamlJobsMatch(self):
+        jobs2 = self.LoadBootstrapYaml('job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml')
+        for name, job in self.LoadBootstrapYaml('job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml').iteritems():
+            if job['repo-name'] is 'k8s.io/kubernetes':
+                job2 = jobs2[name]
+                self.assertEquals(job['max-total'], job2['max-total'])
```

Comment:
It doesn't need to be a one-liner, I just don't want someone to be able to add an attribute to one file without the test catching it.